ci-operator/step-registry/ipi/conf/vsphere/template/ipi-conf-vsphere-template-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: ipi-conf-vsphere-template
```

Comment:
Let us reference the step in `cucushift-installer-rehearse-vsphere-ipi-template-provision`, too.